docker/development.Dockerfile

```diff
@@ -2,7 +2,7 @@
 # 
 # $ docker push typesense/typesense-development:latest
 
-FROM typesense/ubuntu-10.04-gcc:6.5.0
+FROM typesense/ubuntu-10-04-gcc:6.4.0
```

Comment:
Any reason this had to be downgraded?

Reply:
I had initially tagged it as `6.4.0` to refer to the GCC version we were using. However, when you made some changes and republished with `6.5.0` it no longer referred to the gcc version! So, I have moved it back to `6.4.0`.